modules/commons/src/test/java/org/apache/synapse/commons/executors/config/PriorityExecutorFactoryTest.java

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.synapse.commons.executors.config;
+
+import junit.framework.TestCase;
+import org.apache.axiom.om.OMElement;
+import org.apache.axiom.om.util.AXIOMUtil;
+import org.apache.axis2.AxisFault;
+import org.apache.synapse.commons.executors.PriorityExecutor;
+
+import javax.xml.stream.XMLStreamException;
+import java.util.Properties;
+
+public class PriorityExecutorFactoryTest extends TestCase {
+
+    public void testCreateExecutor() throws XMLStreamException, AxisFault {
+        OMElement element = AXIOMUtil.stringToOM("<priority-executor xmlns:ns=\"http://ws.apache.org/ns/synapse\" name=\"executor1\">" +
+                "<ns:queues></ns:queues>" +
+                "</priority-executor>");
+        PriorityExecutor executor = PriorityExecutorFactory.createExecutor("http://ws.apache.org/ns/synapse",
+                element, false, new Properties());
+        assertNotNull("Error creating PriorityExecutor", executor);
+    }
+
+    public void testCreateExecutorWithQueues() throws XMLStreamException, AxisFault {
```

Comment:
shall we add small description Javadoc mentioning what is tested in each testcase

Reply:
Added